src/script/conversation/EventMapper.ts

```diff
@@ -643,6 +651,42 @@ export class EventMapper {
     return messageEntity;
   }
 
+  /**
+   * Maps JSON data of conversation.message_add message into message entity.
+   *
+   * @param event Message data
+   * @returns Content message entity
+   */
+  private _mapEventMultipartAdd(event: MultipartMessageAddEvent) {
+    const {data: eventData} = event;
+    const messageEntity = new ContentMessage();
+
+    const assets = this._mapAssetMultipart(eventData);
+    messageEntity.assets.push(assets);
+
+    if (eventData.text?.quote) {
+      const {message_id: messageId, user_id: userId, error} = eventData.text.quote as any;
+      messageEntity.quote(new QuoteEntity({error, messageId, userId}));
+    }
+
+    return messageEntity;
+  }
+
+  private _mapAssetMultipart(eventData: MultipartMessageAddEvent['data']) {
+    const {text, attachments} = eventData;
+
+    const assetEntity = new Multipart({id: '', text: text?.content || '', attachments});
```

Comment:
why empty id? can't we use createRandomUUID function?

Reply:
I believe it's not needed there, it's the same case as in the text message - `id` is marked as deprecated (look at `EventBuilder.ts`). 

Added it here because it's required in types - we should consider deleting it n the future or change it somehow...